blueocean-web/src/main/js/main.jsx

```diff
@@ -158,9 +161,11 @@ function startApp(routes, stores) {
 
     // Start React
     render(
-        <Provider store={store}>
-            <Router history={history}>{ makeRoutes(routes) }</Router>
-        </Provider>
+          <I18nextProvider i18n={ i18n }>
```

Comment:
why does this have anything to do with react? we should not use react specific stuff here, just make the `t` localization function available globally